sdk/eventhub/tests.yml

```diff
@@ -6,7 +6,7 @@ extends:
       ServiceDirectory: eventhub
       TestTimeoutInMinutes: 240
       BuildTargetingString: azure-eventhub*
-      SupportedClouds: 'Public,UsGov,China'
+      SupportedClouds: 'Public,UsGov'
```

Comment:
Did you intend to remove the China test validation as well?

Reply:
Yes, they've been failing since the switch to Federated Auth, along with USGov. Ben mentioned that his PR should fix this, so I'll add it back it back in. https://github.com/Azure/azure-sdk-for-python/pull/37815